.grit/patterns/migrate_to_v0_31_0.md

```diff
@@ -37,7 +37,7 @@ pattern rewritten_names() {
     `toCustom` => `transform`,
     `toTrimmed` => `trim`,
     `toTrimmedEnd` => `trimEnd`,
-    `toTrimmedStart` => `trimStart`
+    `toTrimmedStart` => `trimStart`,
```

Comment:
Might be good to run `grit patterns test` before merges, since this missing comma breaks the pattern from running at all.